server/Server.Domain/Commands/SetupCustomerCommand.cs

```diff
@@ -25,10 +26,23 @@ CancellationToken cancellationToken
 
         Customer customer = new() { Id = Id, CreationDate = ctx.time.GetUtcNow(), };
 
+        var homelessBookBookshelf = new Bookshelf()
+        {
+            Id = Guid.NewGuid(),
+            Name = "Homeless Books",
+            CreationDate = ctx.time.GetUtcNow(),
+            UpdatedDate = ctx.time.GetUtcNow(),
+        };
+        if (Books is { })
```

Comment:
Not keen on having to do this check twice, made the code more simple in where the database additions is being done. Worth investigating to find a better approach